bin/check_pylint_diff

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Copyright (C) 2016 Google Inc., authors, and contributors <see AUTHORS file>
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+# Created By: miha@reciprocitylabs.com
+# Maintained By: miha@reciprocitylabs.com
+
+set -o nounset
+set -o errexit
+
+ARG1=${1:-}
+TMP_REPO=".temp_repo"
+SCRIPT=$(basename "$0")
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+
+cd "${SCRIPTPATH}/.."
+
+
+if [[ "$ARG1" == "-h" || "$ARG1" == "--help" ]]; then
+  echo "
+Usage: $SCRIPT [test_commit] [-h]
+
+This script will compare pylint error count from two different commits.
+Note: all changes that are not committed will be ignored.
+
+
+Given the commit tree:
+
+       D---E---F---G---H
+            \\         /
+             A---B---C
+
+- Running '$SCRIPT' on C will check the diff between B and C
```

Comment:
Your travis checks are made so it tests the PR, so it will always work on a merge commit. Which means, it will pass if and only if the amount of lint issues will not increase After the PR is merged.